backend/product/src/main/java/org/ecom/product/domain/Product.java

```diff
@@ -17,10 +20,19 @@
 @Data
 @Entity
 public class Product {
-  @Id @GeneratedValue private Long id;
+
+  @Id
+  @GeneratedValue
+  private Long id;
+
+  @Column(name = "category_id")
+  @JsonProperty("category_id")
   private Long categoryId;
-  private String productName;
```

Comment:
I think it's better to name these variables as "name" and "description" than productName and productDesc. The product looks like a redefinition inside the same Product class.